config/src/config/storage_config.rs

```diff
@@ -264,7 +264,7 @@ impl Default for LedgerPrunerConfig {
             // This assumes we have 1T disk, minus the space needed by state merkle db and the
             // overhead in storage.
             prune_window: 150_000_000,
-            batch_size: 500,
+            batch_size: 5_000,
```

Comment:
what is the implication and impact of this change?

Reply:
Would be slightly faster, and less log.